src/DataFrames.jl

```diff
@@ -49,7 +49,6 @@ export AbstractDataFrame,
        nrow,
        nullable!,
        order,
-       printtable,
```

Comment:
This will stop exporting the `printtable` function as suggested. If we go with that then we need to fix the tests to call `DataFrames.printtable()` https://travis-ci.org/JuliaData/DataFrames.jl/jobs/287109648#L606